SwiftCompilerSources/Sources/Optimizer/PassManager/Context.swift

```diff
@@ -256,6 +256,22 @@ struct FunctionPassContext : MutatingContext {
     return false
   }
 
+  func specializeVTable(for type: Type, in function: Function) -> VTable? {
+    guard let vtablePtr = _bridged.specializeVTableForType(type.bridged, function.bridged) else {
+      return nil
+    }
+    notifyInstructionsChanged()
+    return VTable(bridged: BridgedVTable(vTable: vtablePtr))
+  }
+
+  func specializeClassMethodInst(_ cm: ClassMethodInst) -> Bool {
+    if _bridged.specializeClassMethodInst(cm.bridged) {
+      notifyInstructionsChanged()
+      return true
+    }
+    return false
+  }
```

Comment:
I guess this is the same as Erik's comment above. Then this can just be 
```suggestion
  func specializeClassMethodInst(_ cm: ClassMethodInst) -> Bool {
    _bridged.specializeClassMethodInst(cm.bridged)
  }
```

Or 

```suggestion
  func specializeClassMethodInst(_ cm: ClassMethodInst) -> Bool {
    defer { notifyInstructionsChanged() }
    return _bridged.specializeClassMethodInst(cm.bridged)
  }
```

If you still need it for whatever reason.

Reply:
Added notifyCallsChanged().